x-pack/plugins/ml/server/lib/register_settings.ts

```diff
@@ -30,5 +34,43 @@ export function registerKibanaSettings(coreSetup: CoreSetup) {
         }),
       },
     },
+    [ANOMALY_DETECTION_ENABLE_TIME_RANGE]: {
+      name: i18n.translate('xpack.ml.advancedSettings.enableAnomalyDetectionDefaultTimeRangeName', {
+        defaultMessage: 'Enable time filter defaults for anomaly detection jobs',
```

Comment:
Or as this only applies to the results pages, should this be `Enable time filter defaults for anomaly detection results`?

Reply:
Updated here https://github.com/elastic/kibana/pull/76347/commits/1118b3e08345110bec102f86bfbd3bf46cce969c